src/System.Diagnostics.Process/tests/ProcessTests.Linux.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Xunit;
+
+namespace System.Diagnostics.Tests
+{
+    partial class ProcessTests
+    {
+        [Fact]
+        public void TestProcessStartTime_Deterministic_Across_Instances()
```

Comment:
It's enough to add the PlatformSpecific attribute. You don't need a separate file.

Reply:
Done. Allowed it to run on Windows as well since underlying Win32 API GetProcessTimes seems to be deterministic. Not sure what to do with OSX - have no experience